back-end/controllers/record-controller.js

```diff
@@ -39,33 +41,35 @@ const recordController = {
       const recordId = Number(req.params.rid)
       const { title, amount, note, categoryId } = req.body
       const userId = req.user.id
+      console.log(userId)
```

Comment:
Better remove this, I'd approve anyway. :)

Reply:
Oh, noooooo
I didn't notice this line.